FORMAT_ADIOS.md

```diff
@@ -33,27 +33,27 @@ Output from `bpls -A` for a boolean attribute `pybool` stored in the location of
 There is no convention yet for a unique representation of ADIOS2 variables with boolean type.
 Thus, implementations should cast the data to and from `unsigned char` instead.
 
-## `stepBased` Encoding of Iterations
+## `variableBased` Encoding of Iterations
 
-The `iterationEncoding` mode `stepBased` must be implemented via ADIOS steps.
+The `iterationEncoding` mode `variableBased` must be implemented via ADIOS steps.
 
 ## Datasets
 
-An openPMD **data set** is represented by an group prefix that contains an ADIOS variable `__data__`.
+An openPMD **data set** is represented by an ADIOS `Variable` at the location where it would usually be stored.
 
 **attributes** are defined further below and can also appear at the dataset's **group** prefix level.
 
 ## Attributes
 
-openPMD **attributes** stored as ADIOS `Variables` at the location where they would usually be stored.
+openPMD **attributes** stored as ADIOS `Attributes` at the location where they would usually be stored.
 
 Example for a mesh record `E` with record component `x` and attributes `unitDimension` and `unitSI`:
 ```
-  double    /data/meshes/E/unitDimension     10*{7}
-  double    /data/meshes/E/x/__data__        10*{1000}
-  double    /data/meshes/E/x/position        10*{1}
-  double    /data/meshes/E/x/unitSI          10*scalar
+  double    /data/meshes/E/unitDimension          attr   = {0, 0, 0, 0, 0, 0, 0}
+  int32_t   /data/meshes/E/x                      {1000}
```

Comment:
@franzpoeschel that is a bit unusual, since the electric field `E_x` is usually a floating point value
```suggestion
  double   /data/meshes/E/x                      {1000}
```